### PR TITLE
bridge: Move dbus tracking client side

### PR DESCRIPTION
**Pull request by petervo**

### CODE REVIEW

File: pkg/base1/cockpit.js [hunk 1]
```diff
@@ -1598,6 +1606,11 @@ function full_scope(cockpit, $, po) {
                 $.extend(cache.meta, msg.meta);
             } else if (msg.owner !== undefined) {
                 $(self).triggerHandler("owner", [ msg.owner ]);
+
+                if (track && owner)
+                    self.close();
```
Comment (stefwalter) [inline pkg/base1/cockpit.js]: We guarantee that we'll never get an "owner" message with the same owner as previously sent right? If so we should just have a comment about that here.

**Comment by stefwalter** [comment]
This is good in general. We should also implement ```track``` so that we actually communicate only with that unique name, and wait for it before we start sending messages. But maybe that's a follow up commit.

**Comment by petervo** [comment]
I added the comment.

I'm not sure what you mean about enforcing communicating with the unique name, isn't the name set when the channel is setup with no way to change it. Or are you taking about filtering parameters passed to subsequent methods making sure they aren't breaking out of that name?

**Comment by stefwalter** [comment]
> I'm not sure what you mean about enforcing communicating with the unique name, isn't the name set when the channel is setup with no way to change it. Or are you taking about filtering parameters passed to subsequent methods making sure they aren't breaking out of that name?

Well I mean talking to the unique name we first see and never talking to another dbus service for the lifetime of the channel. That was the intent of ```track: true```. But I can implement this part separately. The ```owner``` work will make this pretty easy.

**Pull request closed without merging**